corehq/apps/user_importer/importer.py

```diff
@@ -737,11 +748,20 @@ def process_existing_user(self, user):
                 if self.column_values["profile_name"]:
                     _check_profile(self.column_values["profile_name"], self.domain_info.profiles_by_name)
                     profile = self.domain_info.profiles_by_name[self.column_values["profile_name"]]
+                tableau_role = self.column_values["tableau_role"] if "tableau_role" in self.column_values else None
```

Comment:
I think `self.column_values` would always contain `tableau_role` key since it's added in `_process_column_values`

Reply:
yep, I think that's right. making that change

Reply:
should be the same for `tableau_groups` so I'll skip this check `if "tableau_groups" in self.column_values` also